src/manager/manager.go

```diff
@@ -0,0 +1,242 @@
+package manager
+
+import (
+	"Reactive-Welfare-Housing-System/src/distributor"
+	"Reactive-Welfare-Housing-System/src/messages/distributorMessages"
+	"Reactive-Welfare-Housing-System/src/messages/managerMessages"
+	"Reactive-Welfare-Housing-System/src/messages/propertyMessages"
+	"Reactive-Welfare-Housing-System/src/messages/sharedMessages"
+	"Reactive-Welfare-Housing-System/src/messages/verifierMessages"
+	"Reactive-Welfare-Housing-System/src/property"
+	"Reactive-Welfare-Housing-System/src/storage"
+	"database/sql"
+	"fmt"
+	"log"
+	"reflect"
+	"time"
+
+	"github.com/AsynkronIT/protoactor-go/actor"
+	"github.com/AsynkronIT/protoactor-go/router"
+)
+
+type managerActor struct {
+	db             storage.HouseSystem
+	distributorPID *actor.PID
```

Comment:
Emmm, I guess you mean the struct of distributorActor? Since among manager, property and distributor, only distributor need maintain the cache img so there exist `vacant` and `occupied`

Reply:
Sorry, I marked the wrong code. Forget this comment.